core/modules/node/tests/node.test

```diff
@@ -3407,39 +3408,26 @@ class NodePublishScheduling extends BackdropWebTestCase {
     $node = node_load($node->nid);
     $this->assertEqual($node->status, '1', 'The node is in the state Published.');
 
-    // Ensure scheduled dates without date module also are validated and saved.
-    module_disable(array('date'), FALSE);
-
-    // Without Date module doing the validation, ensure Node module still
-    // checks the date is valid.
+    // Ensure Node module still checks the date is valid.
     $name = $this->randomName(9);
+    $invalid_date = format_date(REQUEST_TIME, 'custom', 'dddY-m-d', $web_user->timezone);
     $edit = array(
       'title' => $name,
       'status' => NODE_SCHEDULED,
-      'scheduled' => format_date(REQUEST_TIME, 'custom', 'dddY-m-d g:i a', $web_user->timezone),
+      'scheduled[date]' => $invalid_date,
+      'scheduled[time]' => format_date(REQUEST_TIME, 'custom', 'H:i', $web_user->timezone),
     );
     $this->backdropPost('node/add/foobsee', $edit, t('Save'));
-    $this->assertText(t('You have to specify a valid date.'));
+    $this->assertText(t('@date is not a valid date.', array('@date' => $invalid_date)));
 
-    // Ensure scheduling in the past is still not allowed without Date module.
+    // Ensure scheduling in the past is not allowed.
     $edit = array(
       'title' => $name,
       'status' => NODE_SCHEDULED,
-      'scheduled' => format_date(REQUEST_TIME - 300, 'custom', 'Y-m-d g:i a', $web_user->timezone),
+      'scheduled[date]' => format_date(REQUEST_TIME - 300, 'custom', 'Y-m-d', $web_user->timezone),
+      'scheduled[time]' => format_date(REQUEST_TIME - 300, 'custom', 'H:i:s', $web_user->timezone),
     );
     $this->backdropPost('node/add/foobsee', $edit, t('Save'));
     $this->assertText(t('Scheduled publish time cannot be in the past.'));
-
-    // Test successful submission and scheduling.
```

Comment:
Not clear why this test was removed.

Reply:
Previously it was necessary to test with both, date module enabled and disabled.
As now the date module isn't involved anymore, this is a duplicate test of the one above:
https://github.com/indigoxela/backdrop/blob/2b5947ea9c2bfa8f2a6112336796977e532656f2/core/modules/node/tests/node.test#L3389